pfcpiface/bess.go

```diff
@@ -471,12 +486,51 @@ func (b *bess) sim(u *upf, method string) {
 
 		fars := []far{farDown, farN6Up, farN9Up}
 
+		// create/delete uplink qer
+		qerDown := qer{
+			qerID: n3,
+			fseID: uint64(n3TEID + i),
+
+			qfi:      9,
+			ulStatus: 0,
+			dlStatus: 0,
+			ulMbr:    50000,
+			dlMbr:    50000,
+			ulGbr:    50000,
+			dlGbr:    50000,
```

Comment:
Pls add following params for per flow End to end metering integration & testing .
For SRTCM
AMBR, cbs,ebs ,
For TRTCM cbs , pbs
look at other params which are required as well- .